### PR TITLE
Validate undefined and null value in criticality column table products

### PR DESCRIPTION
**Pull request by FelixHernandez**
## Validate undefined and null value in criticality column table products
[sc-2297]

**Description**
Now the code validates that `data`  within render function, in criticality column, should have a value. According to datatables documentation:  https://datatables.net/manual/tech-notes/4#Resolution

### CODE REVIEW

File: dojo/templates/dojo/product.html [hunk 1]
```diff
@@ -350,8 +350,16 @@ <h3 class="has-filters">
...
+                      if (type === 'sort') {
+                        return criticals[getDojoExportValueFromTag(data, 'i', 'data-content')]
+                      }
+                      else if (type === 'export') {
+                          return getDojoExportValueFromTag(data, 'i', 'data-content')
+                      }
+                      else if (data != undefined||data != null) {
+                          return data
+                      }
+                      return ""
```
Comment (cneill) [inline dojo/templates/dojo/product.html:362]: Just a minor styling suggestion:

```suggestion
                      if (type === 'sort') {
                        return criticals[getDojoExportValueFromTag(data, 'i', 'data-content')]
                      } else if (type === 'export') {
                          return getDojoExportValueFromTag(data, 'i', 'data-content')
                      } else if (data != undefined||data != null) {
                          return data
                      }
                      return ""
```

Comment (Maffooch) [reply dojo/templates/dojo/product.html:362]: Agreed with this request

Comment (FelixHernandez) [reply dojo/templates/dojo/product.html]: Changes made in this [commit](https://github.com/DefectDojo/django-DefectDojo/pull/8765/commits/cc45195efc05563e006525dc5cdfe537ecf5f7dc)

**Review by cneill — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by grendel513 — APPROVED** [review]

**Review by devGregA — APPROVED** [review]

**Pull request merged**